src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -1401,7 +1410,7 @@ export class BundleDataClient {
       event.inputAmount
     }-${event.outputAmount}-${event.originChainId}-${event.depositId.toString()}-${event.fillDeadline}-${
       event.exclusivityDeadline
-    }-${event.message}-${event.destinationChainId}`;
+    }-${event.messageHash}-${event.destinationChainId}`;
```

Comment:
Can probably eventually swap out this function into a utility function and use it in the SpokePoolClient in lieu of `getDepositHash`

Reply:
We can also use this util in SpokePoolClient [here](https://github.com/across-protocol/sdk/pull/844#discussion_r1936966152) so that we no longer use the `getRelayDataHash()` function  anywhere except when calling `relayFillStatuses()`

Reply:
Can we do this in this PR? That way we can unit test the function as well

Reply:
I ended up pulling it out, but it's already been merged here: https://github.com/across-protocol/sdk/pull/860.